lib/fwdanalysis.h

```diff
@@ -74,9 +74,9 @@ class FwdAnalysis {
     /** Result of forward analysis */
     struct Result {
         enum class Type { NONE, READ, WRITE, BREAK, RETURN, BAILOUT } type;
-        explicit Result(Type type) : type(type), token(nullptr) {}
+        explicit Result(Type type) : type(type) {}
         Result(Type type, const Token *token) : type(type), token(token) {}
-        const Token *token;
+        const Token* token{};
```

Comment:
This should use `= nullptr`.

Reply:
Why? Seems more verbose.

Reply:
But its more explicit to what it would be initialized to.

Reply:
If that is our style, then we should enforce it with some kind of check. Otherwise it's just a preference.

Reply:
For information.. I feel more comfortable with `= nullptr` also but I have no strong opinion.